### ISSUE TITLE
单图训练loss一直无法收敛

### ISSUE THREAD

**Issue by 980044579** [ming71/rotate-yolov3#27]
感谢作者用yolo基于pytorch实现倾斜检测，其中丰富的注释对我有很大帮助。

本人目前运行环境已经搭建完毕，本来想跑通测试代码，但是由于作者提供的模型与配置文件不匹配暂时还未跑通。于是打算优先把训练代码跑通。我考虑是在作者提供的1张HRSC训练集上进行训练，然后再在该图片上测试结果。但是loss始终无法收敛，在1.5左右震荡。我目前只修改了配置文件中的学习率，初始为0.001，训练策略如下：
`scheduler= lr_scheduler.MultiStepLR(optimizer, milestones=[100,200,300,400, 500, 600,700,800, 900], gamma=0.1)` 

![5}$F%KL2N @`M(Y9D174X$Q](https://user-images.githubusercontent.com/23648870/96680679-8adb6680-13a8-11eb-92fb-fd8f59e826ac.png)

目前有几个问题想向作者请教

- 在HRSC训练过程中还有哪些比较重要的参数需要调试

- README.md中展示的军舰检测效果图是旋转Yolov3训练后的测试结果展示图吗

- 是否可以提供一份预训练模型 HRSC+_raw_best.pt 的配置文件

期待可以在百忙中回复一下，祝顺意。

**Comment by ming71** [comment]
单图训练是可以收敛的，只是上数据集效果不好，某些地方被我改出bug忘了保存历史版本导致暂时不可复原了。如果想基于这个代码开展工作，可以参考[ryolo](https://github.com/ming71/yolov3-polygon)，有问题欢迎提问，尽量解答。

**Comment by 980044579** [comment]
感谢回复，如有进展会及时与你交流~
